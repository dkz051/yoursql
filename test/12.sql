CREATE DATABASE db;
USE db;
CREATE TABLE wtable1(key0 INT, key1 INT, tql_values INT NOT NULL, wtf DOUBLE, PRIMARY KEY(key1));
CREATE TABLE wtable2(key0 INT, key1 INT, tql_values INT NOT NULL, wtf DOUBLE, PRIMARY KEY(key1));
INSERT into wtable1(key0, key1, tql_values,wtf)values (134, 406, -20, -0.25);
INSERT into wtable1(key0, key1, tql_values,wtf)values (135, 416, -21, -1.25);
select * from wtable1;
