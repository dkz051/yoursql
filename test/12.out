key0	key1	tql_values	wtf
134	406	-20	-0.250000
135	416	-21	-1.250000
